test/libsolidity/solidityExecutionFramework.h

```diff
@@ -236,6 +239,17 @@ class ExecutionFramework
 	};
 
 private:
+	struct sealEngineInit
```

Comment:
actually in this one we need this. 
as the next line 
m_sealEngine(eth::ChainParams().createSealEngine()),
would require sealengine to be initialized